src/core/psxmem.cc

```diff
@@ -410,6 +445,7 @@ void PCSX::Memory::write8(uint32_t address, uint32_t value) {
             }
         } else if ((page & 0x1fff) >= 0x1f00 && (page & 0x1fff) < 0x1f80 && pioConnected) {
             g_emulator->m_pioCart->write8(address, value);
+        } else if (sendWriteToLua(address, 1, value)) {
```

Comment:
Let's add a `LogClass::HW` log line here to indicate the write has been handled by Lua. Same with the other writes. And maybe the reads too.

Reply:
Added logging for reads and writes